code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -489,3 +489,19 @@
 				/obj/item/circuitboard/computer/apc_control,
 				/obj/item/circuitboard/computer/robotics
 				)
+
+/obj/effect/spawner/lootdrop/twenty_percent_drip_suit
+	name = "20% incredibly fashionable outfit spawner"
+	lootdoubles = FALSE
+
+	loot = list(
+		/obj/item/clothing/under/drip = 20,
+		"" = 80)
```

Comment:
Is this actually how to spawn nothing 80% of the time?

Reply:
Copied it from a 25% cyborg mask spawner further up the lootdrop.dm

Reply:
yeah, refer to the tank spawner too; it works as intended, as jank as it seems